src/audits/AriaRoleNotScoped.js

```diff
@@ -35,11 +35,9 @@ axs.AuditRules.addRule({
          * Checks that this element is in the required scope for its role.
          */
         var elementRole = axs.utils.getRoles(element);
-        if (!elementRole || !elementRole.roles.length)
-            return false;
-        elementRole = elementRole.roles[0];
-        if (!elementRole || !elementRole.valid)
+        if (!elementRole || !elementRole.applied)
             return false;
+        elementRole = elementRole.applied;
```

Comment:
I realise this aspect isn't new here, but I'm still not comfortable with reassigning `elementRole` here. Could we create a new var instead, e.g. `appliedRole`?